pkgs/development/libraries/libunistring/default.nix

```diff
@@ -37,7 +37,7 @@ stdenv.mkDerivation rec {
         ./unistdio/test-u16-vasnprintf3.sh: line 16: ./test-u16-vasnprintf1: No such file or directory
         FAIL unistdio/test-u16-vasnprintf3.sh (exit status: 1)
   */
-  enableParallelBuilding = false;
+  enableParallelBuilding = !doCheck || (with stdenv; hostPlatform!=buildPlatform);
```

Comment:
> Since parallelism bug appears to be in the tests

If only the tests can't handle parallelism, why not disable it just for them with `enableParallelChecking = false`?